demo/MvvmNanoDemo/App.cs

```diff
@@ -11,6 +11,9 @@ protected override void OnStart()
 
             SetUpDependencies();
 
+            AddSiteToDetailPages<WelcomeViewModel>(new MasterDetailData("Welcome"));
```

Comment:
I'm not happy with the naming here. Site contradics the page term used anywhere else in XF.